phivk/gradu.al
# Adds the Github auth POC

Not for merging :) 

To test:
- Ran yarn install
- Add the .env keys shared separately 
- Start the application
- Navigate to /storytellers
  - You should be redirected to the login page (may need multiple login pages? not sure yet)
- Click the Github button and authorize
- You should now be able to see the /storytellers united page.
